changelog

```diff
@@ -130,6 +130,11 @@ Version 1.13.5+dev:
        The blocking behavior can be reenabled by setting pass_allied_units=no
        in the [tunnel] tag.
  * Lua API:
+   * Upgrade to Lua 5.3.3+
+     Consult http://www.lua.org/ for a full change list and updated documenation.
+     * compatibility with Lua 5.0, 5.1 and 5.2 removed
+     * bit32 functions removed
```

Comment:
Good. bit32 can now be done using normal bitshift operators and integer data.

Yes, I plan to do the Markdown document. And, yes, I forgot the change the changelog locally .. on it. I was about 80% through checking for other mainline Lua upgrades needed for deprecated functions when Vultraz had me bump the SDL2 requirement to 2.0.4 everywhere and things went to hell.